example/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { gravity1D, gravity2D, vector as Vector, normNumber } from "../src";
+
+import "./index.css";
+
+const positionAttractor: Vector<number> = [100, 100];
```

Comment:
I think this will make the center of the attractor be the center of the circle instead of the top left corner
```
const positionAttractor: Vector<number> = [150, 150];
```

Reply:
Yep yep, nice.